client/src/components/PostCreationPage.tsx

```diff
@@ -9,53 +9,66 @@ import Stack from '@mui/material/Stack';
 import Divider from '@mui/material/Divider';
 import Input from '@mui/material/Input';
 import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Paper from '@mui/material/Paper'
+import Grid from '@mui/material/Grid';
+import TabContext from '@mui/lab/TabContext';
+import TabList from '@mui/lab/TabList';
+import Tab from '@mui/material/Tab';
+import TabPanel from '@mui/lab/TabPanel';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import { Tags } from '@prisma/client';
 
 const PostCreationPage = (): ReactElement => {
   const navigate = useNavigate();
-  const [title, setTitle]: [string, Function] = useState('');
+  const [title, setTitle]: [string, Function] = useState('# ');
   const [body, setBody]: [string, Function] = useState('');
   const [titleFieldTooltip, setTitleFieldTooltip] = useState(false);
   const [bodyFieldTooltip, setBodyFieldTooltip] = useState(false);
-  const [img, setImg]: [any, Function] = useState();
+  // const [img, setImg]: [any, Function] = useState();
```

Comment:
Can this be removed if it isn't being used?

Reply:
No, this will be used later